### PR TITLE
adding docs about cluster membership revokement

### PR DESCRIPTION
**Pull request by MBishop17**

### CODE REVIEW

File: content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md [hunk 1]
```diff
@@ -133,4 +133,8 @@ You can change the cluster or project role(s) that are automatically assigned to
 
 1. If you want to remove a default role, edit the permission and select **No** from the default roles option.
 
-**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
\ No newline at end of file
+**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
+
+### Cluster Membership Revocation Behavior
+
+When you revoke the cluster membership for a user assigned the **Member** permission, that user [loses their cluster roles](#clus-roles) for the cluster, but [retains their project roles](#proj-roles). In other words, although you have revoked the user's permissions to create new projects within the cluster, the user can still access and manage the clusters and namespaces they've created previously. This functionality is intended to prevent project and namespace owners from being locked out of their own projects and namespaces.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md]: They are being revoked access to the cluster (creating new projects but more importantly the nodes). 

They can still access and manage the _projects_ (not clusters) and namespaces.

File: content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md [hunk 2]
```diff
@@ -133,4 +133,8 @@ You can change the cluster or project role(s) that are automatically assigned to
 
 1. If you want to remove a default role, edit the permission and select **No** from the default roles option.
 
-**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
\ No newline at end of file
+**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
+
+### Cluster Membership Revocation Behavior
+
+When you revoke the cluster membership for a user assigned the **Member** permission, that user [loses their cluster roles](#clus-roles) for the cluster, but [retains their project roles](#proj-roles). In other words, although you have revoked the user's permissions to access the cluster and its nodes, the user can still access and manage the projects and namespaces they've created previously. This functionality is intended to prevent project and namespace owners from being locked out of their own projects and namespaces.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md]: We should clarify that these are assuming that they are assigned at the cluster and project level. These roles only exist if you are a memebr of the cluster or project. 

Let me know if that makes sense.

File: content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md [hunk 3]
```diff
@@ -133,4 +133,10 @@ You can change the cluster or project role(s) that are automatically assigned to
 
 1. If you want to remove a default role, edit the permission and select **No** from the default roles option.
 
-**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
\ No newline at end of file
+**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
+
+### Cluster Membership Revocation Behavior
+
+When you revoke the cluster membership for a user that's explicitly assigned membership to both the cluster _and_ a project within the cluster, that user [loses their cluster roles](#clus-roles) but [retains their project roles](#proj-roles). In other words, although you have revoked the user's permissions to access the cluster and its nodes, the user can still access and manage the projects and namespaces they've created within the cluster previously. This functionality is intended to prevent project and namespace owners from being locked out of their own projects and namespaces.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md]: `and manage the projects and namespaces they've created within the cluster previously`

it's not only projects they created, but projects that they are members of

File: content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md [hunk 4]
```diff
@@ -133,4 +133,13 @@ You can change the cluster or project role(s) that are automatically assigned to
...
-**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
\ No newline at end of file
+**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
+
+### Cluster Membership Revocation Behavior
+
+When you revoke the cluster membership for a user that's explicitly assigned membership to both the cluster _and_ a project within the cluster, that user [loses their cluster roles](#clus-roles) but [retains their project roles](#proj-roles). In other words, although you have revoked the user's permissions to access the cluster and its nodes, the user can still access and manage:
+
+- The projects they hold membership in.
+- The namespaces that they've created.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md]: I'm not sure if the namespaces point is true as auth is tied at the project level so if I created in a namespaces that i think lose project level access, I do not think I would still have access to the namespaces. Can you cerify this?

Comment (MBishop17) [reply content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md]: @cjellick , can you please take a look at this passage and let @deniseschannon and me know if it's accurate?

File: content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md [hunk 5]
```diff
@@ -133,4 +133,13 @@ You can change the cluster or project role(s) that are automatically assigned to
...
+**Result:** The default roles are configured based on your changes. Roles assigned to cluster/project creators display a check in the **Cluster/Project Creator Default** column.
+
+### Cluster Membership Revocation Behavior
+
+When you revoke the cluster membership for a user that's explicitly assigned membership to both the cluster _and_ a project within the cluster, that user [loses their cluster roles](#clus-roles) but [retains their project roles](#proj-roles). In other words, although you have revoked the user's permissions to access the cluster and its nodes, the user can still:
+
+- Access the projects they hold membership in.
+- Exercise any [individual project roles](#project-role-reference) they are assigned.
+
+This functionality is intended to prevent project and namespace owners from being locked out of their own projects and namespaces. If you want to completely revoke a user's access within a cluster, revoke both their cluster and project memberships.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/rbac/cluster-project-roles/_index.md]: we should remove the namespace reference here. basically it's only about projects.

**Pull request merged**